web/themes/custom/pineapple/templates/user/user--account--full.html.twig

```diff
@@ -18,28 +18,88 @@
+            <img src="{{ profile_image_url }}" alt="{{ profile_image_alt }}">
+          </div>
           {% include "@components/_heading.twig" with {
             heading_level: 1,
-            heading_attributes: create_attribute().addClass('author-name'),
+            heading_attributes: create_attribute().addClass('subheading'),
             heading_text: content.field_u_name|field_value|first['#context'].value
           } %}
-          <p>{{ content.field_u_biography|field_value }}</p>
-        </div> <!-- End of author-text -->
-      </div> <!-- End of author-info-container -->
-    </div> <!-- End of author-container -->
+        </div> <!-- End of author-name-container -->
+        <div class="author-text u-o-1">
+          <div class="author-info">
+            {% set user_roles = user.getRoles() %}
+            {% set member_since = user.getCreatedTime()|date("d/m/Y") %}
+            <div class="member-since">
+              <strong>Member since</strong>
+              <p>{{ member_since }}</p>
+            </div>
+            {% if 'recipe_editor' in user_roles or 'article_editor' in user_roles %}
+              <div class="roles">
+                  <strong>Role(s)</strong>
+                {% if 'recipe_editor' in user_roles %}
+                  <p>Recipe Editor</p>
+                {% endif %}
+                {% if 'article_editor' in user_roles %}
+                  <p>Article Editor</p>
+                {% endif %}
+              </div> <!-- End of roles -->
+            {% endif %}
+          </div> <!-- End of author-info -->
+          {% if content.field_u_biography|field_value %}
+            <p>{{ content.field_u_biography|field_value }}</p>
+          {% endif %}
+        </div>
+        {% if 'recipe_editor' in user_roles or 'article_editor' in user_roles or 'administrator' in user_roles %}
+          <div class="sidebar-buttons u-o-1">
+            {% set current_user_id = bamboo_load_currentuser().id %}
+            {% if current_user_id == user.id %}
+              {% set edit_link = '/user/' ~ user.id ~ '/edit'  %}
+              {% include "@components/_link.twig" with {
+                link_text: 'Edit Profile',
+                link_url: edit_link,
+                link_attributes: create_attribute().addClass('btn', 'btn--primary')
+              } %}
+              {% include "@components/_link.twig" with {
+                link_text: 'Create Content',
+                link_url: '/node/add',
+                link_attributes: create_attribute().addClass('btn', 'btn--tertiary')
+              } %}
+            {% endif %}
+          </div>
+        {% endif %}
+      </div> <!-- End of author-container -->
+      <div class="sidebar-links u-o-1">
+        {% include "@components/_heading.twig" with {
+          heading_level: 2,
+          heading_text: 'Useful Links',
+          heading_attributes: create_attribute().addClass('h3')
```

Comment:
Looks like there's some duplication of classes, when this renders there's a subheading and h3 class. You only need one.

Reply:
This is a recurring issue on the site which I am trying to figure out how to solve

Reply:
Done